README.md

```diff
@@ -22,7 +22,10 @@ This Bot Sample was tested with Mattermost server version 3.10.0.
 
```

Comment:
(suggestion) Since step 2 (adding users) depends on the server running, step 2 and 3 makes sense to be swapped.

Also, since the bot account that is created in the `add_users.sh` script is no longer used by `main.go`, it may make sense to change that script too, to not create it.

Reply:
Yeah, fixed